src/styles/index.css

```diff
@@ -7,7 +7,7 @@
 }
 
 body {
-  background-color: rgb(212, 212, 212);
+  background-color: rgb(241, 241, 241);
```

Comment:
- [ ] please make sure to use the right background color in order to match the Design `#fafafa`